ecosystem/node-checker/src/evaluators/metrics/consensus/mod.rs

```diff
@@ -2,7 +2,11 @@
 // SPDX-License-Identifier: Apache-2.0
 
 mod proposals;
+mod round;
+mod timeouts;
 mod types;
 
 pub use proposals::*;
+pub use round::*;
```

Comment:
nit: if you're gonna export everything, why not just declare the modules public above? e.g.,:
```
pub mod proposals;
```

Reply:
I wanted to flatten the imports out, so `evaluators::metrics::MyEvaluator`, not `evaluators::metrics::proposals::MyEvaluator`. In my mind the directory structure below metrics is just for organizing the code, not the modules. But I could be swayed 😛